src/middlewared/middlewared/plugins/failover.py

```diff
@@ -388,7 +389,7 @@ def sync_to_peer(self, options):
         )
 
         if options['reboot']:
-            self.middleware.call_sync('failover.call_remote', 'system.reboot', [{'delay': 2}])
+            self.middleware.call_sync('failover.call_remote', 'system.reboot', 'Failover sync to peer', [{'delay': 2}])
```

Comment:
```suggestion
            self.middleware.call_sync('failover.call_remote', 'system.reboot', SYSTEM_UPGRADE_REBOOT_REASON, [{'delay': 2}])
```

Reply:
Please confirm that these two should not differ